### ISSUE TITLE
Localhost database host UX Setup Issue

### ISSUE THREAD

**Issue by co60ca** [paragonie/airship#54]
- [ ] Check this box if this is a security vulnerability.

### Summary
If you are using localhost as a database host it will give
```
Could not create a database connection. Please check your username and password.
```
if you do not replace the _placeholder_ value of localhost with the word localhost or any loopback.

This is more of a UX issue than a bug IMHO but I think most would assume if the placeholder is there you don't have to fill it in. I suppose your use of "(optional)" on Port should indicate that all other fields are required, but this isn't super user friendly.

### Expected Outcome
It should use localhost anyway, perhaps set the value of database_0_host's input to localhost instead of the placeholder, or set a default on the backend to try localhost if the field isn't set.

### What Actually Happened
Returns to credential screen

### Relevant Debug Information

**Comment by paragonie-scott** [comment]
> perhaps set the value of database_0_host's input to localhost instead of the placeholder, or set a default on the backend to try localhost if the field isn't set.

That should already be happening. I'll look at it right now.

**Comment by co60ca** [comment]
I pulled the master branch before building with docker.

**Comment by paragonie-scott** [comment]
Can you tell me if https://github.com/paragonie/airship/commit/56417c766e140df3c8fb821b1128fafb909f6e86 fixes this for you? `localhost` is the default here.

**Comment by paragonie-scott** [comment]
Off-topic: I wish I could let people select their own tags.

**Comment by co60ca** [comment]
Confirmed the fix in 56417c7
